docs/extensions/ListViewBase.md

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
The ListViewBase extensions are renamed to ListViewExtensions, this doc is unnecessary.